test/angular-selector-test.js

```diff
@@ -0,0 +1,47 @@
+import { AngularSelector, waitForAngular } from '../lib';
+
+fixture `AngularSelector`
+    .page('http://localhost:8080/test/data/angular')
+    .beforeEach(async t => {
+        await t.expect(waitForAngular()).ok({ timeout: 20000 });
+    });
+
+test('root', async t => {
+    const root        = AngularSelector();
+    const rootAngular = await root.getAngular();
+
+    await t
+        .expect(root.exists).ok()
+        .expect(rootAngular.rootProp1).eql(1);
+});
+
+test('selector', async t => {
+    const list        = AngularSelector('list');
+    const listAngular = await list.getAngular();
+
+    await t.expect(list.count).eql(2)
+        .expect(AngularSelector('list-item').count).eql(6)
+        .expect(listAngular.id).eql('list1');
+});
+
+test('composite selector', async t => {
+    const listItem           = AngularSelector('list list-item');
+    const listItemAngular6   = await listItem.nth(5).getAngular();
+    const listItemAngular5Id = listItem.nth(4).getAngular(({ state }) => state.id);
```

Comment:
It seems `getAngular` method is not described in the docs

Reply:
Yes, you are right